src/Exceptions/HttpExceptionHtmlResponse.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @var string $title
+ * @var string $phrase
+ * @var string $href
+ */
+?>
+<!DOCTYPE HTML>
+<html lang='en'>
+<head>
+    <title><?php echo $title ?></title>
+</head>
+<body>
+    <style>
+        * {
+            font-family: Arial, Helvetica, sans-serif;
+            text-align: center;
+        }
+
+        body {
+            background: #1b1c1d;
+            color: white;
+            padding-top: calc(50vh - 95px);
+        }
+
+        body>div {
+            max-width: 90%;
+            margin: auto;
+            width: fit-content;
+        }
+    </style>
+    <div>
+        <h1>
+            <?= $code ?> - <?= !empty($href) ? "<a href='{$href}'>{$phrase}</a>" : $phrase; ?>
```

Comment:
While I still think my previous comment about this applies, it's an improvement I suppose

Except, you cant use single quotes for HTML attributes ;)

Reply:
[HTML Attributes W3 Schools](https://www.w3schools.com/html/html_attributes.asp)

> Double quotes around attribute values are the most common in HTML, but single quotes can also be used. In some situations, when the attribute value itself contains double quotes, it is necessary to use single quotes:

Reply:
Let me rephrase that,

you cant use single quotes and be respected